contracts/SafeMathMock.sol

```diff
@@ -0,0 +1,45 @@
+pragma solidity ^0.4.17;
+
+// Copyright 2017 OpenST Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
```

Comment:
any specific reason for SafeMatchMock contract? Can't we use SafeMatch contract in SafeMath.js directly?